src/auth/auth.spec.ts

```diff
@@ -139,13 +139,17 @@ describe('FirebaseAuth', () => {
       });
     authSpy['getRedirectResult'].and.returnValue(Promise.resolve(null));
 
-    inject([FirebaseApp, AngularFireAuth], (_app: firebase.app.App, _afAuth: AngularFireAuth) => {
-      app = _app;
-      afAuth = _afAuth;
-      authData = null;
-      authCb = null;
-      backend = new FirebaseSdkAuthBackend(app);
-    })();
+    var mockNgZone = Zone.current.fork({
```

Comment:
This code actually doesn't need to be run in a zone. I should fork the zone in the spec that checks the zone to make sure the subscription callback isn't in the `<root>` zone.